Main/Renamer_UpperLower.cpp

```diff
@@ -61,18 +62,40 @@ void Renamer_UpperLower::RenameList(BList *FileList) {
 		char	*tempStr = new char[laenge + 1];
 
 		uint8	Buchstabe, Byte0 = 0;
-		bool	MakeUpper = !bool(fUpperOrLower->Menu()->IndexOf(fUpperOrLower->Menu()->FindMarked()));
+		int32   choice = fUpperOrLower->Menu()->IndexOf(fUpperOrLower->Menu()->FindMarked());
+	
+		int32 isCap = 1;
```

Comment:
Can this be a boolean? For example bool toCapitalize

Reply:
@janus Okay.